app/javascript/components/upload/SequenceFileStep.jsx

```diff
@@ -43,29 +43,29 @@ function SequenceForm({
     <div className="row">
       <div className="col-md-12">
         <div className="form-terra">
-          Primary sequence information, such as BAM, BAM Index, and FASTQ files<br/>
+          Primary sequence information, such as BAM, BAM Index, and FASTQ files<br />
           <p>
-            <b>Non-human Data</b><br/>
+            <b>Non-human Data</b><br />
             If you have a few, small (under 2 GB) non-human sequence files, they can be uploaded here.
             For uploading many or larger files, please refer to the instructions in
-            our <a href="https://singlecell.zendesk.com/hc/en-us/articles/360061006011-Uploading-Large-Files-Using-Gsutil-Tool" target="_blank" rel="noopener noreferrer">documentation</a>.
+            our <a href="https://singlecell.zendesk.com/hc/en-us/articles/360061006011" target="_blank" rel="noopener noreferrer">documentation</a>.
           </p>
           <p>
-            If you already have gsutil installed you can upload files directly using the following command:
+            If you already have the gcloud CLI installed (Sept 2022 release or later) you can upload files directly using the following command:
           </p>
           <pre>
-            gsutil -m cp /path/to/files gs://{formState.study.bucket_id}
+            gcloud storage cp /path/to/files gs://{formState.study.bucket_id}
           </pre>
           <p>
-            <b>Primary Human Data</b><br/>
+            <b>Primary Human Data</b><br />
```

Comment:
Since it's in the changeset:

```suggestion
            <b>Primary human data</b><br />
```

Reply:
5d5213403